api/src/main/java/org/openmrs/module/dhisreport/api/utils/QuarterlyPeriod.java

```diff
@@ -6,7 +6,7 @@
  *  DHIS2 Reporting module is free software: you can redistribute it and/or modify
```

Comment:
there are many deletes in this file just because there where improper indentations in the previous version.